scripts/record-agent-fixtures.sh

```
@@ -0,0 +1,694 @@
+}
+
+# emit_readme <agent> <out_dir> <cli_version> <terminus_verdict> <events_csv> [os_info] [capture_date]
+emit_readme() {
+  local agent="$1" out="$2" ver="$3" term="$4" events="$5"
+  local os_info="${6:-not recorded}" capture_date="${7:-dry-run}"
+  # Agent-specific trust + record-command wording (codex has a hook-trust gate;
+  # opencode does not).
+  local trust_line record_line home_line
+  case "$agent" in
+    codex)
+      trust_line='codex computes a per-hook `trusted_hash` that cannot be hand-written, so the operator pre-trusts the hooks ONCE via `--setup-trust codex` (interactive TUI); persisted `[hooks.state]` trust then lets recording run with NO `--dangerously-bypass-hook-trust` flag.'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)